webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -56,6 +62,7 @@ class BreadcrumbBar extends React.Component {
           items={breadcrumbItems}
           isTitle={isTitle}
           titleReplacement={titleReplacement}
+          data-tut="breadcrumbs_bar"
```

Comment:
Can we call it `data-tutorial`? It took me some time to understand it's about the tutorial.